src/student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age:, classroom:, name: 'unknown', parent_permission: true)
+    super(name, parent_permission, age)
```

Comment:
- [ ] Kindly ensure you are mindful of the parameters and their arrangements. `age` should come before `name` and `parent_permisson` in the `#super` method.

- [ ] Since you are using the colon notation the assignment of parameters will differ also

  ```
  super(age: age, name: name, parent_permission: parent_permission)
  ```

Reply:
🔥 Fixed in student.rb